lib/hanami/assets/helpers.rb

```diff
@@ -682,6 +752,15 @@ def _typed_asset_path(source, ext)
         asset_path(source)
       end
 
+      def _sri?
+        !!self.class.assets_configuration.sri
+      end
+
+      def _sri_value(source, ext)
+        source = "#{ source }#{ ext }" unless source.match(/#{ Regexp.escape(ext) }\z/)
+        self.class.assets_configuration.sri_value(source) unless _absolute_url?(source)
```

Comment:
@jodosha I added an option in `Hanami::Assets::Configuration` for SRI, following the pattern of the `:digest` option. At the moment it is a simple on or off, but I intend for it to take an optional array of SRI algorithms:

```ruby
assets do
  sri :sha256, :sha512
end
```

The intention would be to generate an SRI digest hash with both algorithms.